extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking.tsx

```diff
@@ -29,9 +31,18 @@ function PanelMeasurementTableTracking({
   const [displayStudySummary, setDisplayStudySummary] = useState(
     DISPLAY_STUDY_SUMMARY_INITIAL_VALUE
   );
+  const initialTrackedFilter = trackedStudy
+    ? filterTracked(trackedStudy, trackedSeries)
+    : filterAny;
+  const [measurementFilters, setMeasurementFilters] = useState({
+    measurementFilter: initialTrackedFilter,
+    untrackedFilter: filterNot('measurementFilter'),
```

Comment:
filterNot('measurementFilter') is confusing, i'm not sure what is it

Reply:
Added docs not it.